### DISCUSSION TITLE
A consideration about Panopticon

### DISCUSSION THREAD

**Discussion by morphinestyle** [akeeba/panopticon#492] [General]
A software that doesn't require Docker or complicated installations, that simply unpacks into a folder to work at 100%, and above all, is self-updating with a simple click...: IT'S PRICELESS!
Try to always maintain this ease of use; any software should be simple.

Thank you, Nicholas.

**Comment by nikosdion** [comment]
I always try to follow the KISS rule: Keep It Simple, Stupid. I hate it when I have to divine what the developer intended, or how to do something basic like updating the software. I originally came to Joomla! because it was dead simple. As long as it's possible, I'm going to keep that same spirit of keeping it simple ;)
